MMM-COVID19.js

```diff
@@ -242,17 +250,20 @@ Module.register("MMM-COVID19", {
 		return wrapper
   },
   // sort according to some key and the order could be 'asc' or 'desc'
-  compareValues: function(key, order = 'asc') {
+  compareValues: function(key, order ) {
```

Comment:
@hweigel doing this, you are not letting someone to sort it by 'desc'

Reply:
I moved from a string operator ('asc' / 'desc') to a boolean this.config.orderAscending (declared in line 17), which is then handed over to function compareValues as the second parameter (in lines 85 and 87). 
To sort descending, line 275 now checks now if 'order' is true (ascending) or false (descending).